src/output.rs

```diff
@@ -37,21 +38,23 @@ pub fn handle_hook_error(err: Error, shellpid: u32, silent: bool) -> i32 {
 }
 
 pub fn print_activation(activated: bool, features: HashSet<Feature>) {
-    if activated {
-        if features.len() == 0 {
-            eprint!("\x1b[1;34mactivated {}", SHADOWENV);
+    if is(Stream::Stderr) {
```

Comment:
While we're shedding the bike 😆: 
This would make for a smaller diff and indentation:
```suggestion
    if !is(Stream::Stderr) {
        return;
    }
```